module/zfs/dmu_traverse.c

```diff
@@ -39,7 +39,7 @@
 #include <sys/zfeature.h>
 
 int32_t zfs_pd_bytes_max = 50 * 1024 * 1024;	/* 50MB */
-int32_t ignore_hole_birth = 1;
+int32_t send_holes_without_birth_time = 1;
```

Comment:
Yeah, it built fine in my testbed with the boolean_t change, but when I submitted the PR, the buildbots all failed on the type. Since I couldn't find any ready examples of any other booleans in the existing ZoL codebase for parameters, I opted to leave it int32_t.